ci/deploy-vmck

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -ex
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+export VAGRANT_DOTFILE_PATH=$(mktemp -d --tmpdir VAGRANT_DOTFILE_XXXXXXXX)
+export VAGRANT_DEFAULT_PROVIDER=vmck
```

Comment:
What happens if you remove this line? I suspect vagrant will just use `vmck` as the provider, but maybe I'm wrong.

Reply:
You are right. They aren't necessary.